mlir/include/mlir/Dialect/OpenACC/OpenACCOps.td

```diff
@@ -1145,6 +1145,9 @@ def OpenACC_ParallelOp : OpenACC_Op<"parallel",
     /// The i-th data operand passed.
     Value getDataOperand(unsigned i);
 
+    /// Used to retrieve the block inside the op's region.
+    Block &getBody() { return getRegion().front(); }
```

Comment:
I think in the long term it would be nice to define an interface `OpenACCComputeRegionOp` or smth like that so we make sure all ops implement what we expect. This can be done later.